### PR TITLE
README.md Update

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Changes proposed
Installation Steps
- Clone the project `git clone <url>`
 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1266"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Alternatively, skip all the steps by using [![Gitpod Ready-to-Code](https://img.
 
 ## 👨‍💻 Contributing
 
-- Contributions make the open source community such an amazing place to learn, inspire, and create.
+- Contributions make the open source community such an amazing place to learn, inspire,collaboration and create.
```
Comment (kumarsonsoff3) [inline README.md:52]: There is a missing space and a grammatical error!

```suggestion
- Contributions make the open source community such an amazing place to learn, inspire, collaborate, and create.
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Before installation, please make sure you have already installed the following t
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project `git clone <url>`
```
Comment (vikasganiga05) [inline README.md:40]:
```suggestion
2. Clone the project `git clone https://github.com/<your-github-username>/LinkFree`
```

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Hi, my suggestion is to add the full URL template for the forked repository.

**Comment by emmalearnscode** [comment]
This change only demonstrates cloning with https. I wonder if instead a screenshot would be better of where to find the git clone instruction under the code button?

**Pull request closed without merging**